routes/question.js

```diff
@@ -0,0 +1,33 @@
+exports.record = function(req, res){
```

Comment:
I just realized that we'll need to add a few `require` statements at the beginning of each file to give this file access to the DB schemas as well as resError. I've added those lines to my class.js file in a recent PR. Feel free to take a look at that and add in similar require statements here too!